services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -912,8 +929,26 @@ class HankeServiceITests : DatabaseTest() {
             assertThat(hanke.status).isEqualTo(HankeStatus.DRAFT)
             assertThat(hanke.hankeTunnus).isEqualTo(application.hankeTunnus)
             assertThat(hanke.nimi).isEqualTo(application.applicationData.name)
-            assertThat(hanke.perustaja?.nimi).isEqualTo("Teppo Testihenkilö")
-            assertThat(hanke.perustaja?.email).isEqualTo("teppo@example.test")
+        }
+
+        @Test
+        fun `generates hankekayttaja for founder based on application`() {
+            val inputApplication = AlluDataFactory.cableReportWithoutHanke()
+
+            val application = hankeService.generateHankeWithApplication(inputApplication, USER_NAME)
+
+            val hanke = hankeRepository.findByHankeTunnus(application.hankeTunnus)!!
+            val users = hankeKayttajaRepository.findByHankeId(hanke.id)
+            assertk.assertThat(users.first()).all {
+                prop(HankeKayttajaEntity::id).isNotNull()
+                prop(HankeKayttajaEntity::sahkoposti).isEqualTo(AlluDataFactory.teppoEmail)
+                prop(HankeKayttajaEntity::nimi).isEqualTo(TEPPO_TESTI)
+                prop(HankeKayttajaEntity::permission).isNotNull().all {
+                    prop(PermissionEntity::kayttooikeustaso)
+                        .isEqualTo(Kayttooikeustaso.KAIKKI_OIKEUDET)
+                }
```

Comment:
There's only one test inside all{}:
```suggestion
                prop(HankeKayttajaEntity::permission).isNotNull()
                    .prop(PermissionEntity::kayttooikeustaso)
                        .isEqualTo(Kayttooikeustaso.KAIKKI_OIKEUDET)
                
```